packages/ckeditor5-table/src/utils/ui/table-properties.ts

```diff
@@ -161,6 +161,8 @@ export function getBorderStyleDefinitions(
  */
 export function fillToolbar<TView extends View, TPropertyName extends keyof TView>(
 	options: {
+		isToggleable?: boolean;
+		role?: string;
```

Comment:
Can we name this option, so it's clearer that it's about aria role?

It is a good opportunity to update the API docs of this function.

Nitpick: is it feasible to have string literal union instead of `string` as a type?

Reply:
I dropped these options entirely. It looks like `fillToolbar` creates only radio buttons (as there is single `propertyName`). https://github.com/ckeditor/ckeditor5/pull/17728/commits/51ee7d595ff089f826264e9b5db8bae2beb5f836